modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -3114,6 +3114,42 @@ public void testFilterByStringOperatorsObjectEntriesByRelatesObjectEntriesFields
 			_objectDefinition3);
 	}
 
+	@Test
+	public void testFilterObjectEntriesByRelatesSystemObjectEntriesFields()
+		throws Exception {
+
+		// 1 to many relationship
```

Comment:
Should this be `1 to many... ` or `One to many...` 😁

Reply:
(because before the PR I mean, the comments was // 1 to many...)